ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -92,6 +92,25 @@ def Atanh(*, x, name="Atanh"):
     return ivy.atanh(x)
 
 
+@to_ivy_arrays_and_back
```

Comment:
Please add `@with_supported_dtypes`. You can read about this decorator [here](https://lets-unify.ai/docs/ivy/overview/deep_dive/data_types.html#supported-and-unsupported-data-types).